envsubst/schema.yaml

```diff
@@ -0,0 +1,35 @@
+application_api_version: v1beta1
```

Comment:
Hmm, I'm confused now. Are we moving the source of the e2e-testing solution to somewhere else?